### PR TITLE
Add config.useCropFirst

### PR DESCRIPTION
**Pull request by iBinh**
Added a new config `useCropFirst` property that allows users to choose Crop or Filter will be displayed first in `FMImageEditorViewController`

### CODE REVIEW

**Comment by congnd** [comment]
Seems no problem to me. But it seems that you are not updating the document in README file, can you consider to add a desscriotion for the newly added config?

File: README.md [hunk 1]
```diff
@@ -132,6 +133,11 @@ FMPhotoEditor provides some default crops that will be fit to you.
 Type: `[FMCroppable]?` 
 Default: all crops provided by FMPhotoPicker.
 
+- <a name="ref-use-crop-first"></a>`useCropFirst`    
+An option that indicates which Crop or Filter view should be displayed first in `FMImageEditorViewController`.
```
Comment (congnd) [inline README.md:137]: > An option that indicates which Crop or Filter view should be displayed first in `FMImageEditorViewController`.

Should be: An option that indicates whether the crop menu should be selected by default in the `FMImageEditorViewController`.

**Review by congnd — CHANGES_REQUESTED** [review]

**Comment by congnd** [comment]
And please squash your commits.

**Comment by iBinh** [comment]
Sorry, just updated

**Comment by congnd** [comment]
> And please squash your commits.

@iBinh 
Could you please do this?

**Comment by iBinh** [comment]
Yes, just squashed

**Comment by congnd** [comment]
@iBinh 
Thanks for your contribution.

**Review by congnd — APPROVED** [review]

**Pull request merged**